gazelle/README.md

```diff
@@ -119,6 +119,16 @@ gazelle_python_manifest(
     # the integrity field is not added to the manifest which can help avoid
     # merge conflicts in large repos.
     requirements = "//:requirements_lock.txt",
+	# include_stub_packages: bool (default: False)
+	# If set to True, this flag automatically includes any corresponding type stub packages
+	# for the third-party libraries that are present and used. For example, if you have 
+	# `boto3` as a dependency, and this flag is enabled, the corresponding `boto3-stubs`
+	# package will be automatically included in the BUILD file.
+	#
+	# Enabling this feature helps ensure that type hints and stubs are readily available 
+	# for tools like type checkers and IDEs, improving the development experience and 
+	# reducing manual overhead in managing separate stub packages.
+	include_stub_packages = True
```

Comment:
nit: fix whitespace (tabs were previously used instead of spaces, please check your editor configuration).
```suggestion
    # include_stub_packages: bool (default: False)
    # If set to True, this flag automatically includes any corresponding type stub packages
    # for the third-party libraries that are present and used. For example, if you have 
    # `boto3` as a dependency, and this flag is enabled, the corresponding `boto3-stubs`
    # package will be automatically included in the BUILD file.
    #
    # Enabling this feature helps ensure that type hints and stubs are readily available 
    # for tools like type checkers and IDEs, improving the development experience and 
    # reducing manual overhead in managing separate stub packages.
    include_stub_packages = True
```